src/app/app.component.ts

```
@@ -12,15 +22,76 @@
+        'full_name': token.user.full_name,
+        'user_id': token.user.user_id,
+        'org_id': token.user.org_id,
+        'org_name': token.user.org_name
+      };
+      this.storageService.set('user', user);
+      this.openInAppIntegration(inAppIntegration);
+    });
+  }
+
+  private setupLoginWatcher(): void {
+    console.log('watchers ready');
```

ESLint

[error] Unexpected console statement.

(no-console)